### PR TITLE
Created options in read method

### PR DESCRIPTION
**Pull request by dbemfica**
Created options in read method to finish read response on commands that have multiple responses. Example **monitor-traffic**

### CODE REVIEW

**Comment by EvilFreelancer** [comment]
Hello @dbemfica thanks for contribution!

How to reproduce multiple responses?
Can you provide additional example of usage?

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 # RouterOS API Client
 
-    composer require evilfreelancer/routeros-api-php
+    composer require proner/routeros-api-php
```
Comment (EvilFreelancer) [inline README.md:11]: Seems this should be changed to original group name.

File: composer.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "evilfreelancer/routeros-api-php",
+  "name": "proner/routeros-api-php",
```
Comment (EvilFreelancer) [inline composer.json:2]: Same as in README.md

**Review by EvilFreelancer — COMMENTED** [review]
I've checked your update and don't see updates in tests, can you please implement tests for your changes?

File: README.md (hunk 1 above)
Comment (dbemfica) [reply README.md:11]: Sorry, i changed vendor name for my fork and forgeted back changed before pull request

File: composer.json (hunk 2 above)
Comment (dbemfica) [reply composer.json:2]: Sorry, i changed vendor name for my fork and forgeted back changed before pull request

File: README.md (hunk 1 above)
Comment (EvilFreelancer) [reply README.md:11]: It's okay, just do not forget to fix it before merge :)

**Comment by dbemfica** [comment]
I will close PL and create another

**Pull request closed without merging**